glowing_waffles/visualization/aij_plots.py

```diff
@@ -77,4 +77,24 @@ def seeing_plot(raw_radius, raw_counts, binned_radius, binned_counts, HWHM,
 
     #save plot as png
     if file_name:
-        plt.savefig(f"{file_name}.png")
\ No newline at end of file
+        plt.savefig(f"{file_name}.png")
+
+def predict_ingress_egress(ingress_time, egress_time):
+    """
+    Parameters
+    ----------
+    ingress_time : number
+        the beginning of an exoplanet transit
+
+    egress_time : number
+        the end of an exoplanet transit
+    """
+    ymin,ymax = plt.ylim()
+
+    #create a vertical line at the ingress time and label it
+    plt.vlines(ingress_time,ymin - 0.08, ymax, linestyle = (0, (5, 10)), color = 'red')
```

Comment:
Why is it `ymin - 0.08` instead of `ymin`? Not saying it *shouldn't* be what you have, just curious about the reason. Also, y might want to define a new variable for `ymin - 0.08` instead of repeating it below.

Reply:
I made it ymin - 0.08 because if the label is at ymin it intersects with the line and is not readable. I think that will have to be readjusted for each new plot to make the labels readable. I will make it into a variable.